src/constants.ts

```diff
@@ -24,3 +22,20 @@ export const HEAD_TITLE_SHORT = 'EF Cryptography Research';
 
 // posts
 export const POSTS_DIR = 'src/posts';
+
+// bounties data sources
+export const BOUNTIES_DATA_SOURCE = 'src/bounties-data-source';
```

Comment:
Same deal - can we set this programmatically (based on the directory structure & markdown filenames)?